docker-compose.yml

```diff
@@ -1517,12 +1516,12 @@ services:
         - ${REPO}:${ARCH}-ubuntu-${UBUNTU}-r-${R}
       args:
         arch: ${ARCH}
-        r: ${R}
         base: ${REPO}:${ARCH}-ubuntu-${UBUNTU}-cpp
-        gcc_version: ${GCC_VERSION}
-        tz: ${TZ}
-        r_prune_deps: ${R_PRUNE_DEPS}
+        gcc: ${GCC}
+        r: ${R}
         r_duckdb_dev: ${R_DUCKDB_DEV:-}
+        r_prune_deps: ${R_PRUNE_DEPS}
+        tz: ${TZ}
```

Comment:
This is just `s/GCC_VERSION/GCC/` and then alphabetically ordering, yeah?

Reply:
Right.

Reply:
Thanks